### PR TITLE
add mapAMRClaims property to /idps API

### PR DESCRIPTION
**Pull request by shantanusardal-okta**
## Description:
- Add `mapAMRClaims` property to /idps API
- Yes this will be in monolith version `2023.05.2`

Corresponding OAS PR - https://github.com/okta/okta-oas3/pull/428
Corresponding monolith JIRA - https://oktainc.atlassian.net/browse/OKTA-568576

Local Preview -
Screen Shot 2023-04-20 at 12 00 07 PM

### Resolves:

* [OKTA-593253](https://oktainc.atlassian.net/browse/OKTA-593253)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/idps/index.md [hunk 1]
```diff
@@ -5663,12 +5692,13 @@ Certificate chain description for verifying assertions from the Smart Card.
...
-| subject      | Policy rules to select the Okta sign-in identifier for the IdP User and determine matching rules | [Subject Policy object](#subject-policy-object)           | FALSE    | FALSE    |
+| Property     | Description                                                                                    | DataType                                                  | Nullable | Readonly | Default |
+| ------------ | ---------------------------------------------------------------                                | -------------------------------------------               | -------- | -------- | ------- |
+| accountLink  | Policy rules to link an IdP User to an existing Okta User                                      | [Account Link Policy object](#account-link-policy-object) | FALSE    | FALSE    |         |
+| mapAMRClaims | Determines whether the IDP should map AMR claims from the IDP into the Okta session            | Boolean                                                   | TRUE     | FALSE    | FALSE   |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/idps/index.md:5698]:
```suggestion
| mapAMRClaims | Determines whether the IdP should map AMR claims from the IdP to the Okta session            | Boolean                                                   | TRUE     | FALSE    | FALSE   |
```

**Review by susanharper-okta — APPROVED** [review]
1 small nit - approving to not hold you up

File: packages/@okta/vuepress-site/docs/reference/api/idps/index.md (hunk 1 above)
Comment (shantanusardal-okta) [reply packages/@okta/vuepress-site/docs/reference/api/idps/index.md]: done, attached screenshot from local preview
Screen Shot 2023-04-20 at 12 00 07 PM

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/idps/index.md](https://github.com/okta/okta-developer-docs/blob/c1e359c8907bab56dcb3c60e1e0632c21a298f29/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fidps%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9273359d-e64a-425f-b6f3-d95cf1903c36) | 54  | 79  | 72  | 75  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.024063bd-a27d-4a36-97be-d00c0d2de54a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**